nodejs/templates/tests/test-service.yaml

```diff
@@ -19,4 +19,11 @@ spec:
         echo 'Testing liveness';
         livenessStatusCode=$(wget -S http://$SERVICE_NAME/health/liveness 2>&1 | grep HTTP/ | awk 'END{print $2}');
         test "$livenessStatusCode" = "200"; echo $?;
+    resources:
```

Comment:
Not sure if its worth getting them from values.

Reply:
done